src/django/api/tests/test_polygon_admin_form.py

```
@@ -0,0 +1,299 @@
+        """Create a polygon and register a stub provider referencing it."""
+        self.model_admin = PolygonAdmin(Polygon, AdminSite())
+        self.polygon = PolygonForm(data={
+            'name': 'referenced_boundary',
+            'description': 'A polygon some feature depends on.',
+            'geojson_text': SQUARE_GEOJSON,
+        }).save()
+
+        class StubProvider:
+            """Pretends to be a provider that depends on our polygon."""
+
+            POLYGON_NAMES = ['referenced_boundary']
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)